src/QsFmt/App.Tests/Tests.fs

```diff
@@ -55,6 +55,23 @@ let private CleanResult =
         Error = ""
     }
 
+let private makeOutput verb files =
+    (Seq.length files |> sprintf "%s %i files:" verb)
```

Comment:
(I don't know F#, I can be wrong)
I have an impression that in the fragment `sprintf "%s %i files:" verb)` the string literal `"%s %i files:"` expects two values, but only one is provided (`verb`).

Reply:
Can also be applicable to the _unchanged_ fragment `(eprintfn "%s, %O" input)` in this PR.

Reply:
This is an F# quirk. The last argument to a function is actually the result of the expression found before the `|>`. An equivalent expression might be  `(sprintf "%s %i files:" verb (Seq.length files))` though this is less idiomatic F#, and doesn't work as well with F#'s type inference.